libpam/pam_modutil_sanitize.c

```diff
@@ -107,8 +107,12 @@ close_fds(void)
 	else
 		fd = (int)rlim.rlim_max - 1;
 
+#ifdef HAVE_CLOSE_RANGE
+	close_range(STDERR_FILENO+1, fd, CLOSE_RANGE_UNSHARE);
```

Comment:
`close_range` can fail so its return code has to be checked and a fallback is needed.

When using `close_range`, no need to calculate `fd`, just use `-1U` instead.

To be on the safe side, please check that `CLOSE_RANGE_UNSHARE` is defined.

Reply:
Done